phaseshifts/lib/__init__.py

```
@@ -86,9 +92,15 @@ def _add_windows_dll_dirs():
     and not os.environ.get("PHASESHIFTS_SKIP_COMPILE_ON_IMPORT")
     and not is_module_importable(LIBPHSH_MODULE)
 ):
+    if "CIBUILDWHEEL" in os.environ:
+        print_package_tree()
     try:
         compile_f2py_shared_library(**FORTRAN_LIBS[LIBPHSH_MODULE])
-    except Exception:  # pragma: no cover - best-effort fallback
+    except Exception as err:  # pragma: no cover - best-effort fallback
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)